src/modules/Board.js

```diff
@@ -0,0 +1,5 @@
+export default class Board {
+  constructor() {
+    this.data = [];
+  }
+}
```

Comment:
Great job @abdozayan12 🤝

- [ ] [Optional] I would recommend you use functional components instead of class components.